.github/workflows/doc.yml

```diff
@@ -7,6 +7,14 @@ concurrency:
   group: doc-${{github.ref}}-${{github.event.pull_request.number || github.run_number}}-${{github.workflow}}
   cancel-in-progress: true
 
+env:
+  celer_upload_doc:  >-
+    ${{  github.ref_name == 'develop'
+      || contains(github.ref_name, 'backports/')
+      || contains(github.head_ref, 'release-')
+      || contains(github.event.pull_request.labels.*.name, 'documentation')
```

Comment:
```suggestion
    ${{  github.ref_name == 'develop'             # push into develop 
      || contains(github.ref_name, 'backports/')  # push into backport
      || contains(github.head_ref, 'release-')    # pull request preparing release
      || contains(github.event.pull_request.labels.*.name, 'documentation')
```

Reply:
Alas comments are included as literals so it doesn't work inside this.